### ISSUE TITLE
Wrong storage method

### ISSUE THREAD

**Issue by MJRamon** [ESX-Org/esx_property#10]
When placing an item at house, weapons are stored in `datastore_data` and items at `addon_inventory_items` without any check of place they've been stored. So they are shared across multiply houses that a player owns.
Weapons and items should be stored per house.

**Comment by benzon** [comment]
In general datastor and addon_inventory_items, should be updated, and not do what it dos right now, in general i think a more custom solution would be better for saving items.

**Comment by CruelAlpha** [comment]
This is correct,
specially when running multiple servers, If a user saves 2 waterbottles on server 1 and goes to server 2, the data gets replaced and looses the waterbottles.

**Comment by Duderocktv** [comment]
So what is the fix for this? My players are experiencing this issue with overwriting their data

**Comment by hotatchanh** [comment]
this issue only can fix by redesign database table or design a new one. Each property has its own storage.

**Comment by Ex0r** [comment]
I agree this needs to be improved on. Just at first glance I was thinking, at least for properties, adding an additional column that references the 'name' of the property from the 'properties' table, and passed into the esx_addoninventory:getInventory server event to make sure that only the property you are actually in and accessing is recovered, and only that property is accessed when trying to store something in the inventory.

As a long term, I'd like to rewrite the inventory management so that it doesn't rely on addoninventory as it's a mess. Instead it would use it's own property inventory, like properties_inventory and reference an individual property with it's own storage.
